syntax="proto2";
option cc_generic_services = true;

package peleus_demo.mysql.asyn;

message MysqlAsynRequest {
    required string message = 1;
};

message MysqlAsynResponse {
    required string message = 1;
};

service MysqlAsynService {
    rpc query(MysqlAsynRequest) returns (MysqlAsynResponse);
};
